data/papers

```diff
@@ -0,0 +1,17 @@
+# 常见的查文献的网站
+# Common literature websites 
+
+# cn
+include:wanfang
+include:cnki
+include:cas
+
+# out of cn
+include:elsevier
```

Comment:
for cn, please add to geolocation-cn 
for "out of cn", please add to geolocation-!cn